### DISCUSSION TITLE
Dynamically disabling and enabling buttons?

### DISCUSSION THREAD

**Discussion by PhillyPhoto** [swiftDialog/swiftDialog#353] [Q&A]
Is there a way to enable/disable buttons once Dialog has launched via "echo ... >> /var/tmp/dialog.log" commands (or otherwise)?

**Comment by bartreardon** [comment]
`echo "button1: disable" >> /var/tmp/dialog.log`
`echo "button2: disable" >> /var/tmp/dialog.log`

https://github.com/swiftDialog/swiftDialog/wiki/Updating-Dialog-with-new-content#available-commands
